pyproject.toml

```diff
@@ -27,4 +27,7 @@ urls = {homepage = "https://contentauthenticity.org", repository = "https://gith
 [project.optional-dependencies]
 test = [
   "pytest < 5.0.0"
-]
\ No newline at end of file
+]
+
+[tool.maturin]
+module-name = "c2pa"
```

Comment:
Because we had to update the name above for trusted publishing, now we need this to "fix" the package name.